src/components/Comments/ContainerComment.js

```diff
@@ -182,7 +183,7 @@ export const ReplyComment = ({
   );
 };
 export const ContainerReply = ({children, isGrandchild}) => (
-  <View style={[{borderColor: isGrandchild ? '#fff' : colors.gray1}]}>{children}</View>
+  <View style={[{borderColor: isGrandchild ? COLORS.white : colors.gray1}]}>{children}</View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from hard-coded color to a dynamic color from the theme is a good improvement for consistency and maintainability. However, it seems like there's a typo in your code. You're using `colors.gray1` which is not defined anywhere in this file. I believe you meant to use `COLORS.gray1`. Here's the corrected line:

```diff
- <View style={[{borderColor: isGrandchild ? COLORS.white : colors.gray1}]}>{children}</View>
+ <View style={[{borderColor: isGrandchild ? COLORS.white : COLORS.gray1}]}>{children}</View>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->